### ISSUE TITLE
CA does not work as expected for escaped references (e.g.: ^default)

### ISSUE THREAD

**Issue by kittaakos** [eclipse/xtext-xtend#242]
CA is broken when using with the escaped `^default`.
Version: Xtend IDE 2.12.0.v20170427-1113.
As always, `<|>` means the cursor offset and of course, let me know if this issue requires further clarification.

```
class Foo {

  def static void main(String[] args) {
    val ^default = 1;
    println(^de<|>);
    // Expected: println(^default);
    // Actual: println(^de^default);
  }

}
```

**Comment by cdietrich** [comment]
caused by guard in `org.eclipse.xtext.xbase.ui.contentassist.XbaseProposalProvider.createLocalVariableAndImplicitProposals(EObject, Anchor, ContentAssistContext, ICompletionProposalAcceptor)`

**Comment by cdietrich** [comment]
Fixed in Xtext 2.13
